kustomize/base/balance-reader.yaml

```diff
@@ -26,18 +26,18 @@ spec:
       labels:
         app: balancereader
     spec:
-      serviceAccountName: boa-ksa
+      serviceAccountName: default
       terminationGracePeriodSeconds: 5
       containers:
       - name: balancereader
-        image: gcr.io/bank-of-anthos-ci/balancereader:v0.5.4
+        image: gcr.io/bank-of-anthos-ci/balancereader:v0.5.3
```

Comment:
This (and other places throughout the diff) looks like a regression (in either case, both are wrong, since we're on 0.5.5 now :) )